aspnetcore/signalr/java-client.md

```diff
@@ -69,6 +69,19 @@ SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further detail
 
 This can safely be ignored.
 
+
+## Configure bearer token authentication
+
+In the SignalR Java client, you can configure a bearer token to use for authentication by providing an "access token factory" to the [HttpHubConnectionBuilder](/java/api/com.microsoft.signalr._http_hub_connection_builder?view=aspnet-signalr-java). Use `withAccessTokenFactory` to provide an [RxJava](https://github.com/ReactiveX/RxJava) [Single<String>](http://reactivex.io/documentation/single.html). With a call to [Single.defer](http://reactivex.io/RxJava/javadoc/io/reactivex/Single.html#defer-java.util.concurrent.Callable-), you can write logic to produce access tokens for your client.
+
+```java
+HubConnection hubConnection = HubConnectionBuilder.create("YOUR HUB URL HERE")
+    .withAccessTokenProvider(Single.defer(() -> {
+        // Your logic here.
+        return Single.just("An Access Token");
+})).build();
```

Comment:
```suggestion
    })).build();
```

super nit pick: I generally prefer to align the closing `)`/`}` on the same level as the opening `(`/`{`. Since all these closing symbols are from the `withAccessTokenProvider`, I think it should be another four spaces in.